static/app/views/settings/organizationMembers/organizationMembersList.spec.tsx

```diff
@@ -566,4 +604,77 @@ describe('OrganizationMembersList', function () {
       );
     });
   });
+
+  describe('inviteBanner', function () {
+    it('renders banner with feature flag', function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/missing-members/',
+        method: 'GET',
+        body: missingMembers,
+      });
+
+      const org = TestStubs.Organization({
+        features: ['integrations-gh-invite'],
+      });
+
+      render(<OrganizationMembersList {...defaultProps} organization={org} />, {
+        context: TestStubs.routerContext([{organization: org}]),
+      });
+
+      expect(screen.getByTestId('invite-banner')).toBeInTheDocument();
+      expect(screen.queryAllByTestId('invite-missing-member')).toHaveLength(5);
+      expect(screen.getByText('See all 5 missing members')).toBeInTheDocument();
+    });
+
+    it('does not render banner if lacking org:write', function () {
+      const org = TestStubs.Organization({
+        features: ['integrations-gh-invite'],
+        access: [],
+      });
+
+      render(<OrganizationMembersList {...defaultProps} organization={org} />, {
+        context: TestStubs.routerContext([{organization: org}]),
+      });
+
+      expect(screen.queryByTestId('invite-banner')).not.toBeInTheDocument();
+    });
+
+    it('invites member from banner', async function () {
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/missing-members/',
+        method: 'GET',
+        body: missingMembers,
+      });
+
+      const newMember = TestStubs.Member({
+        id: '6',
+        email: 'hello@sentry.io',
+        teams: [],
+        teamRoles: [],
+        flags: {
+          'sso:linked': true,
+          'idp:provisioned': false,
+        },
+      });
+
+      MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/members/',
+        method: 'POST',
+        body: newMember,
+      });
+
+      const org = TestStubs.Organization({
+        features: ['integrations-gh-invite'],
+      });
+
+      render(<OrganizationMembersList {...defaultProps} organization={org} />, {
+        context: TestStubs.routerContext([{organization: org}]),
+      });
+
+      const inviteButton = screen.queryAllByTestId('invite-missing-member')[0];
+      await userEvent.click(inviteButton);
+      expect(screen.queryAllByTestId('invite-missing-member')).toHaveLength(4);
```

Comment:
Could you add some checks before the button is clicked to confirm there are five missing members initially?